tools/template/module_software_template.sh

```diff
@@ -1,12 +1,18 @@
 
 
+
 declare -A module_options
 module_options+=(
 	["module_template,author"]="@Tearran"
+	["module_template,ref_link"]="@armbian"
```

Comment:
what should go here? Maintainer or maintainer field should also be added?

Reply:
ref_link was hard to clarifi, I was thinking "reference link" would be for 3rd party homepage. if none armbian docs.

Reply:
Hmm, we have doc_link below too. I think this is then doubled. Not sure we need links to armbian documentation as they are auto-included.